presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -148,6 +150,14 @@ private void updateScreen(WarningsPrinter warningsPrinter)
         printQueryInfo(client.currentStatusInfo(), warningsPrinter);
     }
 
+    private String autoFormatMetricValue(String name, long value)
+    {
+        if (name.contains("Nanos")) {
```

Comment:
this looks a bit too hacky. Maybe @shixuan-fan has better suggestion on this?

Reply:
This is only for rendering the values in the debug mode. If a time metric cannot be detected, it will be shown as count which is not too bad. I agree that this is hacky. Another solution is to add a boolean or enum to track the type of each metric. But that is too heavy. We may not want to add that only for better rendering purpose.

Reply:
It seems to me the best way to avoid this hack is to add an enum in `RuntimeMetric`, but it is somewhat heavy (because it relies on the caller to pass this along). The other way is to make each metric an enum, but that seems to introduce more complexity.

Maybe we could consider adding a TODO here so that if there are different formatting needed beyond the current one, we should revisit and pursue the enum way?

Reply:
Sounds good. Added the TODO.